cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1514,6 +1514,12 @@ if(tmva AND cuda)
       set(cuda OFF CACHE BOOL "" FORCE)
     endif()
   endif()
+
+  # CUDA found -- make sure cuda supports the c++ standard we are using
+  execute_process(COMMAND nvcc --std=c++${CMAKE_CXX_STANDARD} --version RESULT_VARIABLE nvxx_miss_cxx_standard OUTPUT_QUIET ERROR_QUIET)
```

Comment:
No `--std=c++${CMAKE_CXX_STANDARD}`, please. This will probably break. For example, for C++17 the option is usually `-std=c++1z` not `--std=c++17`. Also note that it's only a single dash, not two. It's better to use `${CMAKE_CXX_FLAGS}` or something equivalent to that, like `${CUDA_NVCC_FLAGS}` instead.

Reply:
Duly noted and updated :)